modules/apps/account/account-test/src/testIntegration/java/com/liferay/account/service/test/AccountEntryUserRelLocalServiceTest.java

```diff
@@ -267,6 +269,35 @@ public void testAddAccountEntryUserRels() throws Exception {
 		Assert.assertTrue(users.containsAll(accountUsers));
 	}
 
+	@Test
+	public void testAddPersonTypeAccountEntryUserRel() throws Exception {
+		AccountEntry personTypeAccountEntry =
+			AccountEntryTestUtil.addPersonalAccountEntry(
```

Comment:
We should rename `AccountEntryTestUtil.addPersonalAccountEntry` to `addPersonTypeAccountEntry` to keep the naming consistent

Reply:
👍

Reply:
@dejuknow is there already a ticket for making the naming changes for `Personal` => `Person`?

Reply:
@drewbrokke LPS-115423